packages/react-core/src/components/LabelGroup/LabelGroup.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
```

Comment:
So you the `Label` and `LabelGroup` in separate directories here.  @mcarrano  know y had an opinion about the `Chip` and `ChipGroup` being placed in directory  and therefore on the same page for documentation.  Do you feel that same way for the `Label` and `LabelGroup`?

Reply:
Yeah I could see it going either way - combining to mirror Chips/ChipGroup, or staying separate as users may have already been using Labels and be thrown off seeing the docs for them disappear (replaced by LabelGroup). @mcarrano happy to defer to your feedback on this, currently they're listed separately.

Reply:
I could argue either way, but I agree with your reasoning for keeping Label as a thing on it's own. It would be potentially confusing if that went away. I also think that there are many instances when labels will be used by themselves where chips would rarely be used outside of a group. So let's keep separate.

Reply:
Thanks!